### PR TITLE
Upgrade Handler + Migrator for PMTP

### PR DESCRIPTION
**Pull request by AustinoBombino**

### CODE REVIEW

File: x/clp/keeper/migrations.go [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+func (m Migrator) MigrateToVer2(ctx sdk.Context) error {
+	pools := m.keeper.GetPools(ctx)
+	// compute swap prices for each pool
+	for _, pool := range pools {
+		spe := sdk.ZeroDec()
+		spn := sdk.ZeroDec()
+		pool.SwapPriceExternal = &spe
+		pool.SwapPriceNative = &spn
+		m.keeper.SetPool(ctx, pool)
```
Comment (kingpinXD) [inline x/clp/keeper/migrations.go:23]: Might be a good idea to panic here

Comment (Jedi2002) [inline x/clp/keeper/migrations.go:23]: Might be a good iea to panic here

File: app/setup_handlers.go [hunk 2]
```diff
@@ -12,6 +13,7 @@ const releaseVersion = "0.12.0"
 func SetupHandlers(app *SifchainApp) {
```
Comment (Jedi2002) [inline app/setup_handlers.go:13]: releaseVersion should be 0.13.0-rc1

File: app/setup_handlers.go [hunk 3]
```diff
@@ -12,6 +13,7 @@ const releaseVersion = "0.12.0"
...
 	app.UpgradeKeeper.SetUpgradeHandler(releaseVersion, func(ctx sdk.Context, plan types.Plan, vm m.VersionMap) (m.VersionMap, error) {
 		app.Logger().Info("Running upgrade handler for " + releaseVersion)
+		app.ClpKeeper.SetParams(ctx, clptypes.DefaultParams())
```
Comment (Jedi2002) [inline app/setup_handlers.go:16]: can we move the `SetParams` to migrator as well ?

File: x/clp/keeper/migrations.go [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+	keeper Keeper
+}
+
+func NewMigrator(keeper Keeper) Migrator {
+	return Migrator{keeper: keeper}
+}
+
+func (m Migrator) MigrateToVer2(ctx sdk.Context) error {
+	pools := m.keeper.GetPools(ctx)
+	// compute swap prices for each pool
```
Comment (Jedi2002) [inline x/clp/keeper/migrations.go:17]: We should initialize the pmtpEpoch as well . 
Set EpochCounter = 0 and BlockCounter = 0

**Pull request merged**